framework/synapse/notebook/OEA_py.ipynb

```diff
@@ -2,24 +2,10 @@
   "cells": [
     {
       "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "collapsed": true,
-        "jupyter": {
-          "outputs_hidden": false,
-          "source_hidden": false
-        },
-        "nteract": {
-          "transient": {
-            "deleting": false
-          }
-        }
-      },
-      "outputs": [],
       "source": [
         "####### OEA configuration #############\n",
-        "oea_storage_account = 'yourstorageaccount'\n",
-        "oea_keyvault = 'yourkeyvault'\n",
+        "oea_storage_account = 'stoeadablickw3'\n",
```

Comment:
I think we have to keep the 'yourstorageaccount' and 'yourkeyvault' names, it looks like the installers does a find/replace on these strings when setting up the framework resources

Reply:
Thanks for calling this out, I didn't even realize this change got made